src/homepage/howTo/deleted-message-vault.html

```diff
@@ -0,0 +1,300 @@
+                        (<i>retentionPeriod</i> property) by the administrators. It also supports to delete a single message by the message id.
+                    </p>
+                    <p>
+                        Deleted Message Vault API allow to search messages by various criteria base on mail properties.
+                        At the moment, these are supported mail properties for searching:
+                    </p>
+                    <ul>
+                        <li>deletion date(ISO-8601 Date String)
+                            <ul>
+                                <li>supports before or equals operator</li>
+                                <li>supports after or equals operator</li>
+                            </ul>
+                        </li>
+                        <li>delivery date(ISO-8601 Date String)
+                            <ul>
+                                <li>supports before or equals operator</li>
+                                <li>supports after or equals operator</li>
+                            </ul>
+                        </li>
+                        <li>recipients(List of string)
+                            <ul>
+                                <li>supports contains operator</li>
+                            </ul>
+                        </li>
+                        <li>sender(String)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                            </ul>
+                        </li>
+                        <li>has attachment(Boolean)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                            </ul>
+                        </li>
+                        <li>subject(String)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                                <li>supports equal ignore case matching operator</li>
+                                <li>supports contains matching operator</li>
+                                <li>supports contains ignore case matching operator (with US locale)</li>
+                            </ul>
+                        </li>
+                    </ul>
+
+                    <header class="major">
+                        <h2><b>Make Deleted Message Vault available in James</b></h2>
+                    </header>
+
+                    <p>
+                        To do this, you have to create a configuration file <b>deletedMessageVault.properties</b>, then put it into <b>conf</b> directory of James.
+                        There are two available properties you may want to configure:
+                    </p>
+                    <ul>
+                        <li>
+                            <b>urlPrefix</b>: represent for the prefix of namespace Deleted Message Vault will use to store deleted messages.
```

Comment:
For configuration details, we should avoid duplicating stuff and add links to specific configuration pages IMO